src/uniprotkb/components/entry/DiseaseAndDrugsSection.tsx

```diff
@@ -94,9 +96,35 @@ const DiseaseAndDrugsSection = ({
         features={data.featuresData}
         sequence={sequence}
       />
-      <LazyComponent fallback="Variants" rootMargin="50px">
-        <VariationView primaryAccession={primaryAccession} title="Variants" />
-      </LazyComponent>
+      {hasImportedVariants ? (
```

Comment:
Just curious why you opt for `condition ? <foo> : null`  rather than `condition && <foo>`?. I personally find the former a it bit a trickier to comprehend as I need to look for the else on another line.

Reply:
Probably I initially had a variable with a number instead of a boolean, and wanted to avoid rendering "0". I've changed as suggested